setup.cfg

```diff
@@ -36,6 +36,7 @@ tests =
   pytest-cov
   pytest-mpl
   pytest-regtest
+  py
```

Comment:
Minor but its useful to list package dependencies in alphabetical order for ease of reading, however in this instance because you merged `dev` you've now got `py` in there twice so this line can be removed.